test/unittests/utils/test_utils_inputs.py

```diff
@@ -274,7 +274,7 @@ def test_validate_folder_existence(folder_type):
         _validate_folder_existence(1, folder_type)  # noqa
 
     with pytest.raises(
-        ClinicaBIDSError if folder_type == DatasetType.BIDS else ClinicaCAPSError,
+        ClinicaBIDSError if folder_type == DatasetType.RAW else ClinicaCAPSError,
```

Comment:
This test is still broken because of the match. You need to update the following line with:

```python
match=f"The {'BIDS' if folder_type == DatasetType.RAW else 'CAPS'} directory you gave is not a folder.",
```

Reply:
I've juste rebased, ran the tests again and I get no issue without modifying that line. Could you try again ? Thanks !

Reply:
I've just rebased, ran the tests again and they pass without changing the match line since it matches the function `_validate_folder_existence` error message : 
```
    if not directory.is_dir():
        raise (
            ClinicaBIDSError if folder_type == DatasetType.RAW else ClinicaCAPSError
        )(
            f"The {folder_type.value} directory you gave is not a folder.\n"
            "Error explanations:\n"
            f"\t- Clinica expected the following path to be a folder: {directory}\n"
            "\t- If you gave relative path, did you run Clinica on the good folder?"
        )
```
However, I get an error with `test_check_bids_folder_mismatch_with_caps_error` test.

Reply:
It was also a mismatch error so I modified that. I noticed there are currently two ways the error messages are written. With raw/derivative :

```
def test_check_bids_folder_missing_dataset_description_error(tmp_path):
    from clinica.utils.inputs import check_bids_folder

    with pytest.raises(
        ClinicaBIDSError,
        match=re.escape(
            f"The raw directory ({tmp_path}) you provided is missing a dataset_description.json file."
        ),
    ):
        check_bids_folder(tmp_path)
```
or with BIDS/CAPS : 
```
def test_check_bids_folder_mismatch_with_caps_error(tmp_path):
    from clinica.utils.inputs import check_bids_folder

    (tmp_path / "dataset_description.json").touch()
    (tmp_path / "subjects").mkdir()

    with pytest.raises(
        ClinicaBIDSError,
        match=re.escape(
            f"The BIDS directory ({tmp_path}) you provided seems to be a CAPS "
            "directory due to the presence of a 'subjects' folder."
        ),
    ):
        check_bids_folder(tmp_path)

```

Would you also suggest unifying this _ie_ changing everything to raw/derivative ?